Packs/CommonTypes/IndicatorFields/indicatorfield-keyname.json

```diff
@@ -0,0 +1,74 @@
+{
+    "id": "indicator_keyname",
+    "version": -1,
+    "modified": "2022-09-07T08:55:27.075313+03:00",
+    "name": "Key Name",
+    "ownerOnly": false,
+    "cliName": "keyname",
+    "type": "grid",
+    "closeForm": false,
+    "editForm": true,
+    "required": false,
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 2,
+    "hidden": false,
+    "openEnded": false,
+    "associatedToAll": true,
+    "unmapped": false,
+    "unsearchable": true,
+    "caseInsensitive": true,
+    "columns": [
+        {
+            "key": "name",
+            "displayName": "Name",
+            "type": "shortText",
+            "orgType": "shortText",
+            "required": false,
+            "script": "",
+            "width": 150,
+            "isDefault": true,
+            "fieldCalcScript": "",
+            "isReadOnly": false,
+            "selectValues": null
+        },
+        {
+            "key": "data",
+            "displayName": "Data",
+            "type": "shortText",
+            "orgType": "shortText",
+            "required": false,
+            "script": "",
+            "width": 150,
+            "isDefault": true,
+            "fieldCalcScript": "",
+            "isReadOnly": false,
+            "selectValues": null
+        },
+        {
+            "key": "type",
```

Comment:
עדיף Type לפני ה-DATA כמו שזה בעורך עצמו
![image](https://user-images.githubusercontent.com/24267931/191461310-b1f6a918-0665-44e1-a715-b12fef298291.png)

Reply:
We should also put `Type` as a single select with these values:
`REG_NONE,REG_SZ,REG_EXPAND_SZ,REG_BINARY,REG_DWORD,REG_DWORD_LITTLE_ENDIAN,REG_DWORD_BIG_ENDIAN,REG_LINK,REG_MULTI_SZ,REG_RESOURCE_LIST,REG_FULL_RESOURCE_DESCRIPTOR,0REG_RESOURCE_REQUIREMENTS_LIST,1REG_QWORD,REG_QWORD_LITTLE_ENDIAN`